### DISCUSSION TITLE
Send Email Languge Is Not Selected Languge

### DISCUSSION THREAD

**Discussion by yybinpitaya** [sct/overseerr#2362] [Feature Requests]
Send Email Languge Is Not Selected Languge 

Config
![image](https://user-images.githubusercontent.com/73825545/147219401-d57026ae-eb9e-4522-85bf-adcdceeb1850.png)

Email
![image](https://user-images.githubusercontent.com/73825545/147219510-8e4691d3-64ad-4d74-9316-467fbeed4bbd.png)

**Comment by TheCatLady** [comment]
Only the frontend is localized for now. We do plan to localize the backend as well.

**Reply by jpalanca** [reply]
Please! Translated notifications would be amazing in general. Happy to help with Spanish (Spain) if needed.

**Reply by sct** [reply]
Once we add backend localization it will happen similarly to how we do the frontend. Everything will be handled in Weblate so contributions will always be open and welcome :)

**Reply by humalu** [reply]
So glad to know that this is on the road map.
On side of entire backend localization, allowing partically customization with predifined parameters might also be interesting.
